ptbcontrib/ptb_jobstores/ptb_adapter.py

```diff
@@ -53,12 +53,9 @@ def _prepare_job(job: APSJob) -> APSJob:
         # we'll get incorrect argument instead of CallbackContext.
         prepped_job = APSJob.__new__(APSJob)
         prepped_job.__setstate__(job.__getstate__())
-        # Get the tg_job instance in memory
-        # or the one that we deserialized during _reconstitute (first arg in args)
-        if len(job.args) == 1:
-            tg_job = Job._from_aps_job(job)  # pylint: disable=W0212
-        else:
-            tg_job = job.args[0]
+        # Get the tg_job instance in memory or the one that
+        # we deserialized during _reconstitute (second arg in args)
+        tg_job = job.args[1]
```

Comment:
This is almost exactly what [`Job.from_aps_job`](https://github.com/python-telegram-bot/python-telegram-bot/blob/v20.4/telegram/ext/_jobqueue.py#L849-L869) does